applications/app/views/fragments/gallerySlot.scala.html

```diff
@@ -5,22 +5,24 @@
 @import conf.switches.Switches.CommercialSwitch
 @import mvt.CommercialGalleryBannerAds
 
-@additionalSizes = @{
+@commonSizes = @{Seq("1,1", "2,2", "300,250", "fluid")}
+
+@additionalDesktopSizes = @{
```

Comment:
This should probably either be called `desktopSizes` or not include `commonSizes`.